--- ppc64-diag-2.4.0/ela/Makefile.orig	2011-05-02 19:51:45.007928944 +0200
+++ ppc64-diag-2.4.0/ela/Makefile	2011-05-02 19:54:08.461267082 +0200
@@ -76,7 +76,7 @@
 	@$(call install_files,$(CATALOG),744,$(DESTDIR)/etc/ppc64-diag/message_catalog)
 	@$(call install_files,$(CATALOG_REGEX),744,$(DESTDIR)/etc/ppc64-diag/message_catalog/with_regex)
 	@$(call install_man,$(MANPAGES),$(DESTDIR))
-	$(INSTALL) -d -m 755 /var/log/ppc64-diag
+	$(INSTALL) -d -m 755 $(DESTDIR)/var/log/ppc64-diag
 
 clean:
 	@rm -rf $(CMDS) *.o lex.*.c *.tab.c *.tab.h
