--- ppc64-diag-2.6.1/rtas_errd/dump.c.orig	2013-02-08 10:06:13.000000000 +0100
+++ ppc64-diag-2.6.1/rtas_errd/dump.c	2013-03-24 08:04:35.423174347 +0100
@@ -161,7 +161,7 @@
 		goto scanlog_error;
 	}
 
-	out = open(scanlog_filename, O_WRONLY | O_CREAT | O_TRUNC);
+	out = open(scanlog_filename, O_WRONLY | O_CREAT | O_TRUNC, 0600);
 	if (out <= 0) {
 		log_msg(NULL, "Could not open %s for writing, %s", 
 			scanlog_filename, strerror(errno));
